src/user.js

```diff
@@ -0,0 +1,20 @@
+class User {
+  constructor(userData) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.address = userData.address;
+    this.email = userData.email;
+    this.strideLength = userData.strideLength;
+    this.dailyStepGoal = userData.dailyStepGoal;
+    this.friends = userData.friends;
+  }
+
+  getFirstName() {
+    const firstName = this.name.split(' ');
+    return firstName[0];
+  }
+}
+
+if (typeof module !== 'undefined') {
```

Comment:
Still not 100% sure what this line of code does, but glad to see that it's functional and consistent.